okhttp/src/test/java/okhttp3/InformationalResponseCodeTest.kt

```diff
@@ -30,19 +30,22 @@ class InformationalResponseCodeTest {
     recordFrames = true
   }
 
-  private var client = clientTestRule.newClient()
+  private var client = clientTestRule.newClientBuilder()
+    .followRedirects(false)
+    .build()
 
   @Test
   fun test103() {
```

Comment:
Does master have this test?

Reply:
No, this is a manual test, see line 24.  The problem is mockwebserver in 4.x doesn't have support for informational responses.  So tests on master are better.